src/components/AuthenticatedAppHeader/NotificationsPane.jsx

```diff
@@ -35,162 +38,161 @@ function renderDisplayTextBasedOnMessageType(
 
 export default function NotificationsPane({
   anchorEl,
-  setAnchorEl,
   notifications,
   notificationsLoading,
   refreshNotifications,
+  shouldOpen,
+  setShouldOpen,
 }) {
   const intl = useIntl();
+  const history = useHistory();
+  const queryClient = useQueryClient();
   const theme = useTheme();
   const [
     activeCollaborationNotification,
     setActiveCollaborationNotification,
   ] = useState(null);
-  const {
-    markRead,
-    loading: markReadLoading,
-    success: markReadSuccess,
-  } = usePatchNotification();
+  const { markRead } = usePatchNotification();
 
   return (
-    <Popover
-      open={Boolean(anchorEl)}
-      anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
-      PaperProps={{ style: { marginTop: -8 } }}
-      transformOrigin={{ horizontal: 'right', vertical: 'top' }}
-      anchorEl={anchorEl}
-      onClose={() => setAnchorEl(null)}
-    >
+    <div>
       {Boolean(activeCollaborationNotification) && (
```

Comment:
Moved this out of the popover so that the setShouldOpen(false) therein wouldn't also dismiss the dialog.